resources/sts/4.11/enforced/sts_support_permission_policy_1.json

```diff
@@ -0,0 +1,164 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "cloudtrail:DescribeTrails",
+        "cloudtrail:LookupEvents",
+        "cloudwatch:GetMetricData",
+        "cloudwatch:GetMetricStatistics",
+        "cloudwatch:ListMetrics",
+        "ec2-instance-connect:SendSerialConsoleSSHPublicKey",
```

Comment:
This might raise a flag, thoughts @fahlmant ?

Reply:
If we could limit this down to only the nodes we need that would be ideal. The SSH key doesn't last long, but the permission would allow for push anytime
```
Pushes an SSH public key to the specified EC2 instance. The key remains for 60 seconds, which gives you 60 seconds to establish a serial console connection to the instance using SSH. 
```

from https://docs.aws.amazon.com/ec2-instance-connect/latest/APIReference/API_SendSerialConsoleSSHPublicKey.html